### DISCUSSION TITLE
Base path for ws

### DISCUSSION THREAD

**Discussion by lcanavesio** [MichalLytek/typegraphql-nestjs#43] [Q&A]
Hi @MichalLytek . i Use @typegraphql-nestjs 0.4.1
What is the correct way to change the path for ws? When i change my path to other ws connection fail

Ok: '/graphql'
` TypeGraphQLModule.forRoot({
      emitSchemaFile: true,
      installSubscriptionHandlers: true,
      path: '/graphql'
    })
`

 Failed: '/apiv2/graphql'
` TypeGraphQLModule.forRoot({
      emitSchemaFile: true,
      installSubscriptionHandlers: true,
      path: '/apiv2/graphql'
    })
`

In browser see
WebSocket connection to 'ws://10.101.125.116:3008/apiv2/graphql' failed:

**Comment by MichalLytek** [comment]
I think I had a similar issue, so in examples it's just the default path. Maybe it was fixed upstream in some new release. Converting this discussion to a bug report 😉
